styles/searxng/catppuccin.user.css

```
@@ -18,7 +18,7 @@
 ==/UserStyle== */
 
 /* Domains picked from https://searx.space/. */
-@-moz-document domain("https://search.bus-hit.me/"), domain("https://search.inetol.net/")
+@-moz-document url-prefix("https://search.bus-hit.me/"), url-prefix("https://search.inetol.net/")
```

Would this not have worked too?
```suggestion
@-moz-document domain("search.bus-hit.me"), domain("search.inetol.net")
```

---

It would, but not all instances are hosted on a specific subdomain. As I mentioned in the PR description, something like `domain("opnxng.com")` would unintentionally affect all subdomains `*.opnxng.com`; additionally, `domain()` wouldn't work for instances not hosted on a subdomain like `darmarit.org/searx`.

Even though the domains I mentioned aren't default domains in the userstyle, when you add an additional domain in Stylus, the URL type starts off as whatever the URL type of the existing domain is, so using `url-prefix()` for the defaults saves a few clicks when adding domains.